.github/workflows/test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
       postgres:
-        image: postgres:12.3
+        image: postgres:16.6
```

Comment:
16.6 is the version supplied by Google for CloudSQL: https://cloud.google.com/sql/docs/postgres/db-versions